instaclustr/resource_cluster.go

```diff
@@ -637,66 +816,82 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("cluster_id", cluster.ID)
 	d.Set("cluster_name", cluster.ClusterName)
 
-	clusterProvider := make(map[string]interface{}, 0)
-	mapstructure.Decode(cluster.Provider[0], &clusterProvider)
-	processedClusterProvider := processProvider(d, clusterProvider)
-	d.Set("cluster_provider", processedClusterProvider)
-
-	bundles, err := getBundlesFromCluster(cluster)
-	if err != nil {
-		return err
-	}
-
-	if err := d.Set("bundle", bundles); err != nil {
-		return fmt.Errorf("[Error] Error reading cluster: %s", err)
+	if isClusterSingleDataCentre(*cluster) {
```

Comment:
[question] can this be moved to the conditional below?

Reply:
good point! & resolved.